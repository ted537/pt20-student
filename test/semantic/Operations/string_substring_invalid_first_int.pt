using input;
var subStr = "Hello World" / "string" : 5;
