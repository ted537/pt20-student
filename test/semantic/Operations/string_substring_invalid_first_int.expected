   .tFileDescriptor
   .tLiteralInteger
   oEmitValue
   % value emitted 1
   .tFileBind
   .tLiteralAddress
   oEmitDataAddress
   % value emitted 0
   .tStoreInteger
   .tInitialValue
      .tSkipString
      oEmitNullAddress
      % value emitted -32767
      .tStringData
      oEmitValue
      % value emitted 11
      oEmitString
      % value emitted 72
      % value emitted 101
      % value emitted 108
      % value emitted 108
      % value emitted 111
      % value emitted 32
      % value emitted 87
      % value emitted 111
      % value emitted 114
      % value emitted 108
      % value emitted 100
      % value emitted 0
     oEmitNullAddress
     % value emitted -32767
     .tLiteralAddress
     .tFetchChar
     oEmitValue
     % value emitted 16
      .tSkipString
      oEmitNullAddress
      % value emitted -32767
      .tStringData
      oEmitValue
      % value emitted 6
      oEmitString
      % value emitted 115
      % value emitted 116
      % value emitted 114
      % value emitted 105
      % value emitted 110
      % value emitted 103
      % value emitted 0
     oEmitNullAddress
     % value emitted -32767
     .tLiteralAddress
     .tFetchChar
     oEmitValue
     % value emitted 36
     .tLiteralInteger
     oEmitValue
     % value emitted 5
     .tSubstring
     #eTypeMismatch
   .tInitEnd
   .tLiteralAddress
    .tStoreChar
 .tTrapBegin
 .tTrap
 oEmitTrapKind(trHalt)
 % value emitted 0
